CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+## Contributing
+
+If you have a proposal you want to share with us, follow the description below. You only need to follow the installation instructions if you want to contribute something.
```

Comment:
I put the originally requested sentence "You only need to follow the installation instructions below if you want to contribute something." back in and adapted the surrounding text slightly.

Reply:
ok